Makefile

```
@@ -0,0 +1,48 @@
+.PHONY: deploy install cleanup start yarn migrate format test coverage versions standalone-tests
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)